experiments/Marmot/Venn/tikzlibraryvenn.code.tex

```diff
@@ -0,0 +1,174 @@
+% Copyright 2019 by an anonymous marmot
+%
+% This file may be distributed and/or modified
+%
+% 1. under the LaTeX Project Public License and/or
+% 2. under the GNU Public License.
+%
+% See the file doc/generic/pgf/licenses/LICENSE for more details.
+% Big thanks go to JouleV and samcarter for feedback and suggestions!
+\ProvidesFileRCS{tikzlibraryvenn.code.tex}
+\usetikzlibrary{backgrounds,calc}
+\newif\ifvenn@reversed
+\newif\ifvenn@labelmath
+\venn@reversedfalse
+\tikzset{reuse path/.code={%
+    \pgfsyssoftpath@setcurrentpath{#1}}
+}% unfortunately this is *not* the definition of the `use path` key that has
+% been added to the pgfmanual!
```

Comment:
Uhm... What's the fundamental difference here?
https://github.com/pgf-tikz/pgf/blob/97a18d33fb1a791df41768f1709b6e477972b1e6/tex/generic/pgf/frontendlayer/tikz/tikz.code.tex#L535